db/migrate/20201005132433_remove_all_financial_incentive_records.rb

```diff
@@ -7,8 +7,8 @@ def up
 
   def down
     say_with_time "populating finanical incentive" do
-      Subjects::FinancialIncentiveCreatorService.new.execute
-      Subjects::FinancialIncentiveSetSubjectKnowledgeEnhancementCourseAvailableService.new.execute
+      Subjects::FinancialIncentiveCreatorService.new(year: 2020).execute
+      Subjects::FinancialIncentiveSetSubjectKnowledgeEnhancementCourseAvailableService.new(year: 2020).execute
     end
```

Comment:
why are we changing this?

Reply:
That migrate was for recruitment cycle 2019-2020